src/Tribe/Assets.php

```diff
@@ -230,7 +230,7 @@ public function enqueue( $forcibly_enqueue = null ) {
 
 						// If we have a Callable as the Localize data we execute it
 						if ( is_callable( $localize->data ) ) {
-							$localize->data = call_user_func_array( $localize->data, array( $asset ) );
+							$localize->data = call_user_func_array( $localize->data, [ $asset ] );
```

Comment:
Use `$localize->data($asset)` here to avoid the pretty slow `call_user_func_array` call.

Reply:
And, with one argument, it could have just been `call_user_func( $localize->data, $asset );`.